rest/route/service.go

```diff
@@ -110,6 +110,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/projects/{project_id}/test_stats").Version(2).Get().Wrap(checkUser).RouteHandler(makeGetProjectTestStats(sc))
 	app.AddRoute("/projects/{project_id}/task_stats").Version(2).Get().Wrap(checkUser).RouteHandler(makeGetProjectTaskStats(sc))
 	app.AddRoute("/projects/{project_id}/task_reliability").Version(2).Get().Wrap(checkUser).RouteHandler(makeGetProjectTaskReliability(sc))
+	app.AddRoute("/roles").Version(2).Get().Wrap(checkUser).RouteHandler(makeGetAllRolesHandler(sc))
```

Comment:
Is the idea that any user should be able to see all the roles? Is there any reason these might be private?

Reply:
Some roles might be scoped to private projects, so we don't want to reveal the fact that they exist